sdk/formrecognizer/ai-form-recognizer/MIGRATION-v3_v4.md

```diff
@@ -0,0 +1,232 @@
+# Migrating to `@azure/ai-form-recognizer` Version 4
+
+In major version 4, this package introduces a full redesign of the Azure Form Recognizer client library. To leverage features of the newest Form Recognizer service API (version "2021-09-30-preview" and newer), the new SDK is required, and application code must be changed to use the new clients. Similarly, the new major version 4 of the client library cannot be used to communicate with version 2.1 of the service API. To summarize:
+
+- Version 3 of the `@azure/ai-form-recognizer` package _only_ supports Form Recognizer service API version 2.1, and will not receive support for newer (date-based) versions of Form Recognizer.
+- Version 4 (beta) of the package supports service API version "2021-09-30-preview", and future releases of this major version will support newer service API versions as well.
```

Comment:
consider adding a table to describe support like other languages have

Reply:
I'm not so sure the table I see in the other migration guides applies here. We have a much simpler situation in this package where 2.1 (FormRecognizerClient/FormTrainingClient) is the only thing supported in 3.2.0, and the new API is the only thing supported in 4.0.0-beta.1, so it would just be a 2x2 table of package version and API version with "not supported" in two of the cells, so I think the simple description of "this version of the SDK only supports this version of the API, and the previous package only supports the previous API.

Happy to consider alternatives.

Reply:
Ah true, your explanation seems good to me then!